app/change_set_persisters/hyrax/change_set_persister.rb

```diff
@@ -32,16 +34,35 @@ def save_all(change_sets:)
       end
     end
 
-    def buffer_into_index
-      metadata_adapter.persister.buffer_into_index do |buffered_adapter|
-        with(metadata_adapter: buffered_adapter) do |buffered_changeset_persister|
-          yield(buffered_changeset_persister)
-        end
+    def delete_all(change_sets:)
+      change_sets.map do |change_set|
+        delete(change_set: change_set)
       end
     end
 
-    def with(metadata_adapter:)
-      yield self.class.new(metadata_adapter: metadata_adapter, storage_adapter: storage_adapter)
+    def delete_self_and_members(change_set:)
```

Comment:
Is this a public api?  can we document it?

Reply:
Doc Added